generator/generator.js

```diff
@@ -0,0 +1,73 @@
+var fs = require('fs');
+var parse = require('csv-parse');
+var transform = require('stream-transform');
+
+var output = [];
+var parser = parse({delimiter: ','})
+var input = fs.createReadStream('test.csv');
```

Comment:
This generator,js was my first attempt to parse the csv of prices but I haven't got it working right yet. Its using pipes and streams but I couldn't get it to do the average calculation. For now parser.js will do the same job but its not as reusable. I've deleted this generator.js for now. This commit was a mistake.